### ISSUE TITLE
Submit clifm to Debian unstable

### ISSUE THREAD

**Issue by leo-arch** [leo-arch/CliFM#291]
It would be really nice to have **clifm** in the Debian repos. I've already submitted a [request](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1031287) like a year ago but got no answer.

**Comment by dase78** [comment]
I am a complete enduser but I also would like to see it in "my" repos!

**Comment by leo-arch** [comment]
And we're finally [there](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1031287#25)!

**Comment by dase78** [comment]
This is great, looking forward for it!

**Comment by leo-arch** [comment]
Hi @dase78. It is it indeed! Maybe you'll have to wait a bit for it, since it is has been added to sid yesterday, and it may take some time to hit testing, and then a bit more for stable.

BTW, my thanks to @krekhovx for his work packaging **clifm**.

**Comment by dase78** [comment]
Yes, of course. I am also using Stable but I think, I'll use the Sid-Package then. Anyways, good news.

**Comment by krekhovx** [comment]
Yes, congratulations, we did a great job!
